CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ if(ES_FLATPAK)
 	list(APPEND VCPKG_MANIFEST_FEATURES "flatpak-libs")
 elseif(ES_STEAM)
 	list(APPEND VCPKG_MANIFEST_FEATURES "steam-libs")
+	add_compile_definitions("WAYLAND_DISABLE")
```

Comment:
Please use `target_compile_definitions` instead, and this belongs in source/CMakeLists.txt. I would also call the macro something with steam like ES_STEAM or something so that it can be used for other Steam stuff in the future.